client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -470,6 +471,10 @@ class EmailProvidersComparison extends React.Component {
 		);
 	}
 
+	goBack = () => {
+		page( `/email/${ this.props.selectedSiteSlug }` );
```

Comment:
We can use a [helper function](https://github.com/Automattic/wp-calypso/blob/16a727db49f88d12fd1a13b24c6d010544c89cfb/client/my-sites/email/paths.js#L105) instead of hardcoding the path of this page (here is [an example](https://github.com/Automattic/wp-calypso/blob/6b8e45d00474d63b64891d9a1261fd644a6024b1/client/my-sites/email/email-management/titan-manage-mailboxes/index.jsx#L68)).